[2024-04-12T18:54:20.525+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:54:16.843621+00:00 [queued]>
[2024-04-12T18:54:20.538+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:54:16.843621+00:00 [queued]>
[2024-04-12T18:54:20.539+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 2
[2024-04-12T18:54:20.555+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): insert_qdrant> on 2024-04-12 18:54:16.843621+00:00
[2024-04-12T18:54:20.561+0000] {standard_task_runner.py:60} INFO - Started process 428 to run task
[2024-04-12T18:54:20.570+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', '12345678', 'insert_qdrant', 'manual__2024-04-12T18:54:16.843621+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/pipeline.py', '--cfg-path', '/tmp/tmp8_vt_aom']
[2024-04-12T18:54:20.578+0000] {standard_task_runner.py:88} INFO - Job 40: Subtask insert_qdrant
[2024-04-12T18:54:20.645+0000] {task_command.py:423} INFO - Running <TaskInstance: 12345678.insert_qdrant manual__2024-04-12T18:54:16.843621+00:00 [running]> on host 5feba290cccc
[2024-04-12T18:54:20.736+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='12345678' AIRFLOW_CTX_DAG_ID='12345678' AIRFLOW_CTX_TASK_ID='insert_qdrant' AIRFLOW_CTX_EXECUTION_DATE='2024-04-12T18:54:16.843621+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-04-12T18:54:16.843621+00:00'
[2024-04-12T18:54:20.795+0000] {python.py:202} INFO - Done. Returned value was: {'status': 'success', 'message': 'Data inserted to Vector DB successfully'}
[2024-04-12T18:54:20.814+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=12345678, task_id=insert_qdrant, execution_date=20240412T185416, start_date=20240412T185420, end_date=20240412T185420
[2024-04-12T18:54:20.843+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-04-12T18:54:20.852+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
